### DISCUSSION TITLE
Repo in data

### DISCUSSION THREAD

**Discussion by moraesfernanda** [greenelab/lab-website-template#88] [Q&A] [answered]
Hello,
I am adding resources and tools but could not understand the repo field. What does it do? Where do I find the source code?

Here goes one example:

- name: II Congresso Nacional de Ligas de Anatomia Humana 2021 - Minicurso Neuroimagem
  group: events
  image: images/conlaah.png
  link: https://youtu.be/vPBOveiEbBQ
  description: Minicurso de neuroimagem para o II Congresso Nacional de Ligas de Anatomia Humana 2021. A doutorando Fernanda de Moraes apresentou técnicas de Investigação não-invasiva da morfometria cerebral por imagens de Ressonância Magnética e aplicações.
  repo: greenelab/lab-website-template

Thank you very much in advance and sorry if this is too basic :)

**Comment by vincerubinetti** [comment, marked as answer]
The `repo` field just refers to a GitHub repository that you want to import tags/topics from. For example, this template itself is a repository -- `greenelab/lab-website-template` -- and its tags can be seen on its main github page:

![image](https://user-images.githubusercontent.com/8326331/127376851-93c55e05-8633-40bf-a61a-5940185e91d7.png)

The `repo` field is completely optional. It looks like you're using the resources page to list events, and events probably wont have repos associated with them, so you can just delete those `repo` lines.

---

This is a little confusing, and could be explained better in the documentation. The template can take a `.yaml`. list of pretty much anything, and render it with [any of our many components](https://github.com/greenelab/lab-website-template/wiki/Components). If you ctrl+f the components page for "repo", you'll see an explanation of it under the Card component. The Card component is probably what you're using to display your events list on the Resources page.

I hope this helps.

Perhaps I'll add more of an explanation in [this section](https://github.com/greenelab/lab-website-template/wiki/Basic-Editing#data-and-collections) where I talk about making the list `.yaml` files. What do you think?
